packages/server/lib/project-base.ts

```diff
@@ -813,15 +828,15 @@ export class ProjectBase<TServer extends ServerE2E | ServerCt> extends EE {
 
   // These methods are not related to start server/sockets/runners
 
-  async getProjectId () {
+  async getProjectId (config?: Cfg): Promise<string> {
```

Comment:
```suggestion
  async getProjectId (): Promise<string> {
```

Reply:
We should not need to pass around `config` anymore, just call `getConfig()` - this should be the source of truth.